sycl/test-e2e/ESIMD/lsc/lsc_block_load_store_stateless_flag_64.cpp

```diff
@@ -0,0 +1,85 @@
+//===-lsc_block_load_store_stateless_64 - DPC++ ESIMD on-device test-===//
```

Comment:
This line is copy-pasted from lsc_block_load_store_stateless_64.cpp without adjustment to this file (adding "_flag" suffix).

IMO, having this test in a separate file is a bit overkill - files differ only in 2 lines where flag is passed.
Maybe have templated function test that would accept 'bool TestFlag' ?

Reply:
fixed in latest commit, thx